ui/base/BUILD.gn

```diff
@@ -650,12 +650,21 @@ jumbo_component("base") {
 
   if (use_aura) {
     # Aura clipboard.
-    if (use_x11 && is_desktop_linux) {
-      sources += [
-        "clipboard/clipboard_aurax11.cc",
-        "clipboard/clipboard_aurax11.h",
-        "clipboard/clipboard_linux.cc",
-      ]
+    if (is_desktop_linux) {
+      if (use_x11 || use_ozone) {
+        sources += [ "clipboard/clipboard_linux.cc" ]
```

Comment:
will this file be needed for say ozone/win (something google was upstreaming at some point)?

Reply:
well, this is not platform specific and can be used with mac or win in theory.